### PR TITLE
[INFINITY-1654][KAFKA] Set default and configurable update plan strategy

### PR DESCRIPTION
**Pull request by gabrielhartmann**

### CODE REVIEW

File: frameworks/kafka/universe/config.json [hunk 1]
```diff
@@ -41,10 +41,16 @@
...
             "type":"string",
-            "default":"serial",
-            "enum": ["serial", "serial-canary", "parallel-canary", "parallel"]
+            "enum": ["serial", "serial-canary", "parallel", "parallel-canary"],
+            "default":"parallel"
+          },
+          "update_strategy":{
+            "description":"Strategy to use for rolling out updates {serial, serial-canary, parallel-canary, parallel}",
+            "type":"string",
+            "enum": ["serial", "serial-canary", "parallel", "parallel-canary"],
```
Comment (ArtRand) [inline frameworks/kafka/universe/config.json]: See my comment from the C* PR, basically why not have `serial-canary` be the default?

Comment (NimaVaziri) [inline frameworks/kafka/universe/config.json]: Having a `parallel` (and `parallel-canary`) update plan will introduce downtime, I'm assuming that wasn't intended here?

Comment (gabrielhartmann) [inline frameworks/kafka/universe/config.json]: no change in behavior.

Comment (gabrielhartmann) [inline frameworks/kafka/universe/config.json]: It is intended.  You want the ability to choose this option in emergencies.

**Pull request closed without merging**